### ISSUE TITLE
StatusIcon active/inactive label can not be adjusted by translators on a per-use basis

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#4288]
### Observed behavior

As in #3882, since "active" and "inactive" are adjectives which generally refer to some noun, in certain languages it might be important to adjust the words e.g. for gender.

### Expected behavior

~~Tooltips~~ Labels [updated 2018-10-16] should match the things they describe

### User-facing consequences

In some languages might hinder comprehension. Also, using proper grammar helps learners and makes the app more professional.

### Context

0.11

**Comment by radinamatic** [comment]
I was under the impression something did change here, but it's still impossible to properly translate this `Active/Inactive` label in languages with gendered adjectives.

When viewing the list of Lessons, I adjusted the string to match the required gender in Spanish `Activa/Inactiva`:

![lessons](https://user-images.githubusercontent.com/1457929/47049702-5cfe2e80-d19e-11e8-90d1-264b18567f64.png)

However, navigate to the list of exams, and now they appear with incorrect label `Activa/Inactiva` instead of `Activo/Inactivo`.

![exams](https://user-images.githubusercontent.com/1457929/47049826-b5cdc700-d19e-11e8-985c-25f9ed85cf0c.png)

If I correct them to match the required gender, labels for lessons will be incorrect, and the vicious circle goes on... 😞 

Labels are much more visible than tooltips, and I would argue that this issue is not "minor". 
@indirectlylit Is this possible to fix for 0.11 without breaking the string freeze?

**Comment by indirectlylit** [comment]
> I was under the impression something did change here

We fixed it in the 'change assignment' modal but not here.

> Labels are much more visible than tooltips

Agreed, and it seems I worded this issue incorrectly. I don't know why I thought this was about tooltips and not visible labels.

> Is this possible to fix for 0.11 without breaking the string freeze?

Possible, but slightly difficult. I think the easiest solution is create two variations of the component (`ExamStatusIcon` and `LessonStatusIcon`) with four independent label texts:

* `ExamStatusIcon.active`
* `ExamStatusIcon.inactive`
* `LessonStatusIcon.active`
* `LessonStatusIcon.inactive`

Then in Crowdin we can manually copy over those four translations in every language from:

* `ManageExamModals.changeExamStatusActive`
* `ManageExamModals.changeExamStatusInactive`
* `ManageLessonModals.changeLessonStatusActive`
* `ManageLessonModals.changeLessonStatusInactive`

-----

(as an aside, this is making me realize that ours standard practice of applying and auto-approving translation memory is likely causing a slight decrease in quality due to exact matches in English that aren't matches in other languages)

**Comment by radinamatic** [comment]
Thank you @indirectlylit, it would be great if we could squeeze this in! 👍 
Let me know if I can help.

> ours standard practice of applying and auto-approving translation memory is likely causing a slight decrease in quality due to exact matches in English that aren't matches in other languages

This might just be the case... On a similar note, when new version comes in, not once I found old(er) versions of translated strings that I have corrected or tightened up in the meantime, and that might be the result of applying the translation memory that includes older translations. Is there like a condition statement for that step that would go like "apply the TM translation *ONLY* if it is more recent than the currently approved one"... 🤔
